### PR TITLE
[Backport-1.x] Added release drafter for one click release of SDK

### PR DESCRIPTION
**Pull request by owaiskazi19**
### Description
Backport of https://github.com/opensearch-project/opensearch-sdk-java/pull/837

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by owaiskazi19** [comment]
Let's wait for @gaiksaya to review before merging this as I have changed the version in build.gradle.

**Review by gaiksaya — APPROVED** [review]

**Review by dbwiddis — APPROVED** [review]

**Pull request merged**
